shared/profile/wot-author/index.tsx

```diff
@@ -130,45 +133,78 @@ const VerificationChoice = (props: {
   selected: boolean
   onSelect: () => void
 }) => {
-  let text = <Kb.Text type="Body">Do not choose this option</Kb.Text>
+  const textStyle = Styles.globalStyles.flexOne
+  let text = (
+    <Kb.Text type="Body" style={textStyle}>
+      Do not choose this option
+    </Kb.Text>
+  )
   let color: string = Styles.globalColors.white
   switch (props.verificationType) {
     case 'in_person':
       text = (
-        <Kb.Text type="Body">
+        <Kb.Text type="Body" style={textStyle}>
```

Comment:
I'm unconvinced by all this `text = <Kb.Text....` business.

How about 
```
let text: React.ReactNode
...
text = <>{props.voucheeUsername} told me their username <Kb.Text type="BodyBold">in person</Kb.Text></>
...
text = `${props.voucheeUsername} told me their username over video`
...
etc
...
label={<Kb.Text type="Body" style={Styles.globalStyles.flexOne}>{text}</Kb.Text>}
```

Reply:
fragment to the rescue! That looks better.